src/Test/Data/Environment.ts

```diff
@@ -1,4 +1,35 @@
-import { EnvironmentExpertOnly, EnvironmentModel } from "@/Core";
+import { EnvironmentExpertOnly, EnvironmentModel, FlatEnvironment } from "@/Core";
+
+export const env: FlatEnvironment = {
```

Comment:
Is it right, that you will add a test later for this then? Or where is this env imported :D